src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -175,27 +182,28 @@ let main argv =
 
     printInfo "Registered %d analyzers from %d dlls" analyzers dlls
 
-    let projOpt = results.TryGetResult <@ Project @>
+    let projOpts = results.TryGetResult <@ Project @>
 
     let results =
-        async {
-            match projOpt with
-            | None ->
-                printError
-                    "No project given. Use `--project PATH_TO_FSPROJ`. Pass path relative to current directory.%s"
-                    ""
-
-                return None
-            | Some proj ->
-                let project =
-                    if Path.IsPathRooted proj then
-                        proj
-                    else
-                        Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, proj))
-
-                let! results = runProject client toolsPath project ignoreFiles
-                return results |> Option.map (printMessages failOnWarnings)
-        }
-        |> Async.RunSynchronously
+        match projOpts with
+        | None
+        | Some [] ->
+            printError "No project given. Use `--project PATH_TO_FSPROJ`. Pass path relative to current directory.%s" ""
+
+            [||]
+        | Some projects ->
+            let runProj (proj: string) =
+                async {
+                    let project =
+                        if Path.IsPathRooted proj then
+                            proj
+                        else
+                            Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, proj))
+
+                    let! results = runProject client toolsPath project ignoreFiles
+                    return results |> Option.map (printMessages failOnWarnings)
+                }
+
+            projects |> List.map runProj |> Async.Sequential |> Async.RunSynchronously
```

Comment:
Any reason we don't want to run these in parallel?

Reply:
Yes, I'm worried about mangled console output.